langchain/llms/ctransformers.py

```diff
@@ -103,3 +107,36 @@ def _call(
             text.append(chunk)
             _run_manager.on_llm_new_token(chunk, verbose=self.verbose)
         return "".join(text)
+
+    async def _acall(
+        self,
+        prompt: str,
+        stop: Optional[List[str]] = None,
+        run_manager: Optional[AsyncCallbackManagerForLLMRun] = None,
+        **kwargs: Any,
+    ) -> str:
+        """Asynchronous Call out to CTransformers generate method.
+        Very helpful when streaming (like with websockets!)
+
+        Args:
+            prompt: The prompt to pass into the model.
+            stop: A list of strings to stop generation when encountered.
+
+        Returns:
+            The string generated by the model.
+
+        Example:
+            .. code-block:: python
+                response = llm("Once upon a time, ")
+        """
+        text_callback = None
+        if run_manager:
+            text_callback = partial(run_manager.on_llm_new_token, verbose=self.verbose)
```

Comment:
@agola11 if i understand correctly, we don't need to pass self.verbose in to run manager anymore right? since relevant callback handlers are added during Chain init if verbose is set?